aiohttp_sse/__init__.py

```diff
@@ -138,7 +138,12 @@ async def send(
             buffer.write(self._sep)
 
         buffer.write(self._sep)
-        await self.write(buffer.getvalue().encode("utf-8"))
+        try:
+            await self.write(buffer.getvalue().encode("utf-8"))
+        except ConnectionResetError:
+            if self._ping_task is not None:  # pragma: no cover
```

Comment:
Don't think there should be a no cover here. I think we should just `assert` it, as it should never happen. Alternatively, maybe just call .stop_streaming().